src/main.cpp

```diff
@@ -16,6 +16,9 @@ int main(int argc, char *argv[])
     QCoreApplication app(argc, argv);
     if (argc > 2 && !strcmp(argv[1], "--removeUserFiles"))
         return SailfishUserManager::removeUserFiles(argv[2]);
+    if (argc == 3 && !strcmp(argv[1], "--session")) {
```

Comment:
If we choose to support these scripts here, I would prefer this to be something more like "--runSessionStartScripts".